### DISCUSSION TITLE
Making sketch public using API

### DISCUSSION THREAD

**Discussion by NofLevi10root** [google/timesketch#3229] [General]
How to make sketch public or at least share it with all users using the API ?
Prefer python API client but can also work using normal API request.

**Comment by jkppr** [comment]
For clarification: Do you want to change the sharing settings using the API client? Or do you want to make the Sketch available for all users of your instance to access it via the API client?

**Reply by jkppr** [reply]
If you want to make the Sketch public using the API client, you can use the `sketch.add_to_acl()` function.
https://github.com/google/timesketch/blob/master/api_client/python/timesketch_api_client/sketch.py#L468
